README.md

```diff
@@ -1,2 +1,23 @@
-# Currency-Xchange
-Simple currency converter website
+Currency-Xchange
+
+Currency-changer is a website to enable users to quickly doing their currency exchanging for any country they want to any another
+
+Our Team
+
+Alaa Alser
+Alaa Sahloub
+Sondos Alalami
+
+Why
+Currencies are continuously changing everyday and even every moment , and it's get harder to keep updating it ,so we did this to give rapidly Currency changing
+
+What
+Our website is a single page that enables users to quickly change currencies.
+
+How
+We are using an API called "rest countries", its have all countries information for flags and currencies and do it for both We are using an API called "rest countries", its have all countries information for flags and currencies and do it for both countries,also we are using unsplash API to change background for the country user wants change to .
+
```

Comment:
great but there are repeated sentences. if you can add more details about that the user will search for two countries and their currencies will be displayed and then the conversion, with entered number.